src/gui/MItem_menus.cpp

```diff
@@ -402,17 +402,37 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
     : WI_LABEL_t(_(label), nullptr, is_enabled_t::yes, is_hidden_t::no) {
 }
 
+struct SettingsLoadState {
+    bool is_network : 1,
+        is_connect : 1;
+};
```

Comment:
Tiny nitpick: I think this is the only place in the whole codebase that declares two fields in one statement in a struct; we prefer to repeat the type, eg:

```
struct SettingsLoadState {
   bool is_network;
   bool is_connect;
}
```

Also, having a struct for this is kind of complete overkill ‒ I know you're trying to save 1 byte, but it's on stack and this is probably not the deepest stack around. You can just have them as two variables on stack and everything will be fine (and more readable 😇).

Reply:
I had the custom filament PR in mind, but true, it would also only add another bool and nothing more. I've changed it.